src/vmm/src/seccomp_filters/mod.rs

```diff
@@ -6,6 +6,12 @@ use std::fs::File;
 
 const THREAD_CATEGORIES: [&str; 3] = ["vmm", "api", "vcpu"];
 
+// This byte limit is passed to `bincode` to guard against a potential memory
+// allocation DOS caused by binary filters that are too large.
+// This limit can be safely determined since the maximum length of a BPF
+// filter is 4096 instructions and Firecracker has a finite number of threads.
+const DESERIALIZATION_BYTES_LIMIT: Option<u64> = Some(100_000);
```

Comment:
How was 0.1MB obtained?

Reply:
basically trial and error with a filter length of 4096 instructions and 3 thread categories. using a binary search type approach. The real needed value was a tiny bit smaller, but I added some extra padding just to be sure.

In reality, in the case of Firecracker, the filters are much smaller (they are not 4096 instructions long), but this acts as a memory allocation safe-guard and accounts for the event that the filters are growing close to he kernel limit